products/datastore/terraform.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Index: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "projects/{{project}}/indexes/{{index_id}}"
+    self_link: "projects/{{project}}/indexes/{{index_id}}"
+    import_format: ["projects/{{project}}/indexes/{{index_id}}"]
+    autogen_async: true
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 10
+      delete_minutes: 10
+    examples:
```

Comment:
The sweeper is currently assuming that indexes have a "name" field, so you'll want to either disable or fix the sweeper.

Reply:
Disabled!